addons/website/static/src/js/menu/edit.js

```diff
@@ -330,9 +330,13 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
 
     _getContentEditableAreas () {
         return $(this.savableSelector).not('input, [data-oe-readonly],[data-oe-type="monetary"],[data-oe-many2one-id], [data-oe-field="arch"]:empty').filter((_, el) => {
-            return !$(el).closest('.o_not_editable').length;
+            return !$(el).closest('.o_not_editable, .oe_website_sale .products_header').length;
```

Comment:
Don't mention other modules in website, this should be moved in adequate modules (if the PR remains, I just quickly read it vertically)